cartography/intel/github/util.py

```diff
@@ -70,6 +70,14 @@ def call_github_api(query: str, variables: str, token: str, api_url: str) -> Dic
         # Add context and re-raise for callers to handle
         logger.warning("GitHub: requests.get('%s') timed out.", api_url)
         raise
+
+    if response.status_code == 502:
+        logger.error(
+            "GitHub API returned 502 Bad Gateway error. This often indicates GitHub's internal issues. "
+            "Response headers: %s, Response body: %s",
+            dict(response.headers),
+            response.text,
+        )
```

Comment:
The `response.raise_for_status()`will log downstream the exception, but I agree that logging the body and headers will give more details on the actual error to troubleshoot.

Reply:
@cmm-lyft can you provide an example log message created by this PR? I'm somewhat concerned with how large the message will be when logging a whole set of response headers.